docs/Developer/SequenceDiagrams/graphitti.puml

```diff
@@ -3,8 +3,8 @@
 title Graphitti General Program Flow
 
 == Initialization ==
-Driver -> Simulator: Load Simulator Parameters
-Driver -> Simulator: Instatiate Simulator Objects
+Core -> Simulator: Load Simulator Parameters
+Core -> Simulator: Instatiate Simulator Objects
```

Comment:
Let's fix the typo; should be "Instantiate".

Reply:
My bad, fixed the typo.